packages/tenant-process/test/db.test.ts

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable functional/no-let */
+/* eslint-disable functional/immutable-data */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+import { beforeAll, afterEach, describe, expect, it } from "vitest";
+import { GenericContainer } from "testcontainers";
+import { PostgreSqlContainer } from "@testcontainers/postgresql";
+import {
+  ReadModelRepository,
+  TenantCollection,
+  initDB,
+} from "pagopa-interop-commons";
+import { IDatabase } from "pg-promise";
+import { config } from "../src/utilities/config.js";
+import {
+  ReadModelService,
+  readModelServiceBuilder,
+} from "../src/services/readModelService.js";
+import {
+  TenantService,
+  tenantServiceBuilder,
+} from "../src/services/tenantService.js";
+
+describe("database test", () => {
+  let tenants: TenantCollection;
+  let readModelService: ReadModelService;
+  let tenantService: TenantService;
+  let postgresDB: IDatabase<unknown>;
+  beforeAll(async () => {
+    const postgreSqlContainer = await new PostgreSqlContainer("postgres:14")
+      .withUsername(config.eventStoreDbUsername)
+      .withPassword(config.eventStoreDbPassword)
+      .withDatabase(config.eventStoreDbName)
+      .withCopyFilesToContainer([
+        {
+          source: "../../docker/event-store-init.sql",
+          target: "/docker-entrypoint-initdb.d/01-init.sql",
+        },
+      ])
+      .withExposedPorts(5432)
+      .start();
+
+    const mongodbContainer = await new GenericContainer("mongo:4.0.0")
+      .withEnvironment({
+        MONGO_INITDB_DATABASE: config.readModelDbName,
+        MONGO_INITDB_ROOT_USERNAME: config.readModelDbUsername,
+        MONGO_INITDB_ROOT_PASSWORD: config.readModelDbPassword,
+      })
+      .withExposedPorts(27017)
+      .start();
+
+    config.eventStoreDbPort = postgreSqlContainer.getMappedPort(5432);
+    config.readModelDbPort = mongodbContainer.getMappedPort(27017);
+    tenants = ReadModelRepository.init(config).tenants;
+    readModelService = readModelServiceBuilder(config);
```

Comment:
```suggestion
    const readModelRepository = ReadModelRepository.init(config);
    ({ tenants } = readModelRepository);
    readModelService = readModelServiceBuilder(readModelRepository);
```

Can you make this change in order to prevent the Warning from duplicated connections?